lib/AST/ASTPrinter.cpp

```diff
@@ -1008,19 +1005,23 @@ void PrintAST::printAttributes(const Decl *D) {
     // clients, or if it inherits superclass convenience initializers,
     // then print those attributes specially.
     if (auto CD = dyn_cast<ClassDecl>(D)) {
-      if (Options.PrintImplicitAttrs) {
```

Comment:
@benlangmuir This is all nested inside `if (Options.PrintImplicitAttrs)` up above, so I just eliminated the unnecessary scope.